src/blocks/remote-data-pagination/edit.tsx

```diff
@@ -12,20 +13,17 @@ import './editor.scss';
 
 export function Edit( props: BlockEditProps< RemoteDataPaginationBlockAttributes > ): JSX.Element {
 	const { context } = props;
-	const blockProps = useBlockProps();
-
 	const { remoteData } = useRemoteDataContext( context );
+	const blockProps = useBlockProps();
 
-	if ( ! remoteData?.pagination ) {
+	if ( ! remoteData?.blockName ) {
 		return (
-			<div { ...blockProps }>
-				<Placeholder
-					label={ __( 'Remote Data Pagination' ) }
-					instructions={ __(
-						'This block only works when placed inside a remote data block using data that supports pagination. This block will be ignored as currently configured.'
-					) }
-				/>
-			</div>
+			<Placeholder
```

Comment:
This should be added above the existing check for pagination data, not replace it

Reply:
Do you mean it should be placed as a seperate check to see if its outside the remote data block vs if pagination is configured or not?

Shouldn't this work just like the empty results block where a customer can configure it always in the editor, and then it should only render if there is pagination data?

Reply:
> Shouldn't this work just like the empty results block where a customer can configure it always in the editor, and then it should only render if there is pagination data?

The key difference is that pagination is at the query schema level, so it's known in advance and not a run-time discovery. So we can let the user know it will never work.

Reply:
Fair point. I'll update this to have two separate checks then.